### DISCUSSION TITLE
2FA is not working for logging into my account

### DISCUSSION THREAD

**Discussion by gregoftheweb** [community/community#130442] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### GitHub Feature Area

Security and Privacy

### Body

Hello,
I am unable to login with 2FA.

Steps to repro:
1. login using one of my recovery codes
2. Go to settings>>Passwords and Authentication >> 2FA section >> Authenticator app
3. expand authenticator section
4. Open MS authenticator app on my mobile
5. Add new account on MS Auth app
6. scan QR code --> RESULT >> success -> github added.

NEXT
1. logout of gitub
2. re-login
3. get to prompt where it asks for 2FA 6 digit code
4. Open github account in MS Auth app --> FIND 6 digits
5. Enter 6 digit code in github form field >> click submit

Expected result:
Successful login

Actual result:
failed login.

What am I doing wrong??

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @gregoftheweb ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by gregoftheweb** [comment]
Anybody have any suggestions?

What am I doing wrong?

**Comment by gregoftheweb** [comment]
As Always its the developers fault.  RESOLVED

On the Security page where you scan the 2Fa QR code (which I did), there is one more step there where you enter a one time code from your Auth app on your phone and submit it to confirm its working.  I had not done that step... stupid.  So I did it now and it works.
